packages/edit-site/src/components/post-fields/index.js

```diff
@@ -139,7 +134,7 @@ function PostAuthorField( { item } ) {
 	);
 }
 
-function usePostFields( viewType ) {
+function usePostFields() {
```

Comment:
Nice change, it means we can probably move all these fields into the fields package from now on.